Tutorial Part I (#1050)
iotaledger/iota.js: documentation/docs/tutorials/value-transactions/02-prepare-the-dev-env.md

      you can spin up your own node to run the PoW remotely.

* Access to a Stardust Node (Hornet 2.0.0). You can use the Shimmer testnet Nodes
  at [https://api.testnet.shimmer.network](https://api.testnet.shimmer.network).
* (Optional) A TypeScript compiler and related packages

## (optional) Download the Code

You
can [download the tutorials codebase](https://github.com/iotaledger/iota.js/tree/feat/stardust/packages/iota/examples/shimmer-value-transaction-tutorial)
to follow while you read the next sections. 

:::warning Running Code Examples

If you plan on using the provided code example, please read the article [How To Run Code Examples](../../how_tos/run_how_tos.mdx).

:::

:::warning Shimmer Addresses and Outputs

The tutorial's codebase uses several predefined Shimmer addresses and outputs. If you are going to use other addresses, you will need to update the code accordingly. Likewise, the tutorial codebase assumes a particular state of the addresses and outputs. Before executing the code, please ensure your addresses and outputs are in the expected state.

:::

### Available Commands

> 

Reviewer: ```suggestion

#### Generate Addresses

You can run the `generate-addresses` script by running the following command from the example's directory:

```bash
npm run address
```

#### Send a Value Transaction

You can run the `send-value-transaction` script by running the following command from the example's directory:

```bash

> npm run start

Reviewer: ```suggestion
npm run transaction
```

```

> #### Sweep Funds

Reviewer: ```suggestion
#### Sweep Outputs
```

> You can run the `send-value-transaction` script by running the following command from the example's directory:

Reviewer: ```suggestion
You can run the `sweep-outputs` script by running the following command from the example's directory:
```

```bash
npm run sweep
```

> ## Create Your package.json file

Reviewer: ```suggestion
## Create Your package.json File
```
